README.md

```diff
@@ -51,7 +51,7 @@ The application is a fully functional app that allows users to sign-up/log-in, c
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- Coming soon when the project is finished
+- [Live Demo](https://blue-ray-pay.onrender.com)
```

Comment:
- [ ] Nice job for deploying your app! However, I'm unable to successfully signup/login to the deployed app. I get the below error showing in the browser console whiles attempting to signup. Kindly ensure to improve upon this aspect. You can also consider other [free] hosting sites for fullstack apps such as _'railway.app'_

**Signup/Login error:**
![image](https://user-images.githubusercontent.com/12745474/219696489-ad24dbf6-c7aa-41fe-bf36-dea938758982.png)

Reply:
resolved! ✔️